actions/build-push-to-dockerhub/action.yaml

```diff
@@ -24,7 +24,7 @@ inputs:
   file:
     description: |
       The dockerfile to use.
-    default: "Dockerfile"
+    required: false
```

Comment:
what happens now there's no default: we pass an empty string as the input to the build-push-action and it uses its default?